File: php/Mahasiswa.php
<?php
class Mahasiswa
{
    private $nama; // atribut nama
    private $nim; // atribut nim 
    private $prodi; //atribut prodi
    private $fakultas; // atribut fakultas

    // constructor kelas Mahasiswa
    public function __construct($nama = '', $nim = '', $prodi = '', $fakultas = '')
    {
        $this->nama = $nama;
        $this->nim = $nim;
        $this->prodi = $prodi;
        $this->fakultas = $fakultas;
    }

    // setter nama
    public function setNama($nama)
    {
        $this->nama = $nama;
    }

    // setter nim
    public function setNim($nim)
    {
        $this->nim = $nim;
    }

    // setter prodi
    public function setProdi($prodi)
    {
        $this->prodi = $prodi;
    }

    // setter fakultas
    public function setFakultas($fakultas)
    {
        $this->fakultas = $fakultas;
    }

    // getter nama
    public function getNama()
    {
        return $this->nama;
    }

    // getter nim
    public function getNim()
    {
        return $this->nim;
    }

    // getter prodi
    public function getProdi()
    {
        return $this->prodi;
    }

    // getter fakultas
    public function getFakultas()
    {
        return $this->fakultas;
    }

}

?>

File: python/Mahasiswa.py
# deklarasi kelas Mahasiswa
class Mahasiswa:

    # constructor
    def __init__(self, nama, nim, prodi, fakultas):
        self.__nama = nama
        self.__nim = nim
        self.__prodi = prodi
        self.__fakultas = fakultas

    # setter nama
    def setNama(self, nama):
        self.__nama = nama

    # setter nim
    def setNim(self, nim):
        self.__nim = nim

    # setter prodi
    def setProdi(self, prodi):
        self.__prodi = prodi

    # setter fakultas
    def setFakultas(self, fakultas):
        self.__fakultas = fakultas

    # getter nama
    def getNama(self):
        return self.__nama

    # getter nim
    def getNim(self):
        return self.__nim

    # getter prodi
    def getProdi(self):
        return self.__prodi

    # getter fakultas
    def getFakultas(self):
        return self.__fakultas
